LICENSE.txt

```diff
@@ -0,0 +1,214 @@
+LICENSE
+
+VMware vmw-guestinfo
+
+Copyright � 2017 VMware, Inc.  All rights reserved.
+
+This product is licensed to you under the Apache License version 2.0 (the "License").  You may not use this product except in compliance with the License.
+
+                                 Apache License

```

Comment:
I copied this file from the photon repo and nuked the latter half of it where it mentioned packages they rely on.  I was being lazy :-).  Ill grab the standard file so long as Apache 2.0 is agreeable to you.  I was spitballing :-).